examples/sglang/utils/sglang.py

```
@@ -14,13 +14,16 @@
 # limitations under the License.
 
 import argparse
+import logging
 from argparse import Namespace
 
 from sglang.srt.server_args import ServerArgs
```

Pylint (3.3.7)

[error] No name 'srt' in module 'sglang'

(E0611)